tests/phpunit/api/v4/Action/ContactGetTest.php

```diff
@@ -228,6 +228,8 @@ public function testRegexpOperators(): void {
       ->execute()->indexBy('id');
     $this->assertCount(1, $result);
     $this->assertArrayHasKey($jane['id'], (array) $result);
+
+    // TODO - Add REGEXP BINARY and NOT REGEXP BINARY tests here, maybe
```

Comment:
@agileware-justin did you have a specific question about this test? It seems fairly straightforward, you just call the api and check the result, exactly in the same way it's being done in the above few lines. This does seem like a good place to test the new operators.